sale_order_global_stock_route/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,2 @@
+For use this module you have to activate the option Settings > Inventory
+> Warehouse > Multi-Step Routes
```

Comment:
I think it should contain the following comment:

In the user profile you must select this option "Enable Route on Sales Order Line" in the "Technical Settings" section